arches_project/arches_project.py

```diff
@@ -55,7 +64,7 @@
 import sys
 import requests
 from datetime import datetime
-
+import time
```

Comment:
This looks unused

Reply:
I commented on this before I noticed that there were several other unused imports.